Allow using image directive with non-local images

humitos opened the pull request:
This PR supports the usage of this directive,

    .. image:: https://path.to/image.png

It won't prefix that full URL with language and version.

stsewd commented on notfound/utils.py:
```diff
@@ -25,6 +26,10 @@ def replace_uris(app, doctree, nodetype, nodeattr):
             if uri.startswith('../'):
                 uri = uri.replace('../', '')
 
+        if re.match('^https?://', uri):
```
nit: re.match already match only from the beginning, `^` isn't necessary

humitos replied:
I think it's better to be explicit.

humitos commented:
> Would be good to have a test for an uri that looks like an external url `something/http-protocol-spec.html`

Added this test and fixed a bug that was skipping all the nodes if one was external.
